keyboards/cxt12e4/keymaps/default/config.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 Ismaïl Senhaji (@iSma)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_DISABLE_WHEN_USB_SUSPENDED true
```

Comment:
No true/false.  Just defined or not. 
```suggestion
#define RGB_DISABLE_WHEN_USB_SUSPENDED
```